quality_of_service_demo/rclpy/quality_of_service_demo_py/incompatible_qos.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import sys
+
+from quality_of_service_demo_py.common_nodes import Listener
+from quality_of_service_demo_py.common_nodes import Talker
+
+import rclpy
+from rclpy.duration import Duration
+from rclpy.executors import SingleThreadedExecutor
+from rclpy.logging import get_logger
+from rclpy.qos import QoSDurabilityPolicy
+from rclpy.qos import QoSLivelinessPolicy
+from rclpy.qos import QoSProfile
+from rclpy.qos import QoSReliabilityPolicy
+from rclpy.qos_event import PublisherEventCallbacks
+from rclpy.qos_event import SubscriptionEventCallbacks
+from rclpy.qos_event import UnsupportedEventTypeError
+
+
+def get_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        'incompatible_qos_policy_name',
+        type=str,
+        choices=['durability', 'deadline', 'liveliness_policy', 'liveliness_lease_duration',
+                 'reliability'],
+        help='The QoS Policy that should be incompatible between the publisher and subscription.')
+    return parser
+
+
+def main(args=None):
+    # Argument parsing and usage
+    parser = get_parser()
+    parsed_args = parser.parse_args()
+
+    # Configuration variables
+    qos_policy_name = parsed_args.incompatible_qos_policy_name
+    qos_profile_publisher = QoSProfile(depth=10)
+    qos_profile_subscription = QoSProfile(depth=10)
+
+    if qos_policy_name == 'durability':
+        print('Durability incompatibility selected.\n \
+Incompatibility condition: publisher durability kind < \
+subscripition durability kind.\n \
+Setting publisher durability to: VOLATILE\n \
+Setting subscription durability to: TRANSIENT_LOCAL\n')
```

Comment:
This is really hard to read, I would recommend this other style:
```python3
        print(
            'Durability incompatibility selected.\n'
            'Incompatibility condition: publisher durability kind < '
            ...
        )
```

Reply:
That comment applies in other places too.

Reply:
Done.

Reply:
Thanks!